front/src/modules/ui/layout/board/components/BoardColumn.tsx

```diff
@@ -52,6 +43,32 @@ const StyledNumChildren = styled.div`
   width: 16px;
 `;
 
+const StyledButtonGroup = styled.div`
```

Comment:
@Kanav-Arora Could you actually use the LightIconButton group instead of creating this style